### ISSUE TITLE
Hyperparameter Tuning (Feed-Forward Network / Fully Connected NN)

### ISSUE THREAD

**Issue by Kushal-ghimire** [Subashcoder/Price_Prediction_and_Optimization#15]
1. Parameter selection
2. Dropout selection
3. Model Integrity

**Comment by Nischal156** [comment]
Hyperparameter tuning for feed-forward neural networks involves optimizing parameters like learning rate, architecture, batch size, and regularization to improve accuracy and prevent overfitting,  using random search.
